### ISSUE TITLE
Broken scan icon in Edge

### ISSUE THREAD

**Issue by netandreus** [Authenticator-Extension/Authenticator#330]
**Describe the bug:** In Microsoft Edge scan icon (second icon in top right corner of modal window) is broken. It does not look like other browsers.

**Platform:** 
 - Browser: Edge
 - Browser Version:  44.18362.1.0, EdgeHTML 18.18362
![edge-screen-0](https://user-images.githubusercontent.com/313477/57849854-17ade800-77e5-11e9-98ac-61342aaa704c.png)
![chrome-firefox-screen-0](https://user-images.githubusercontent.com/313477/57849863-1e3c5f80-77e5-11e9-91ec-2ca84ba84975.PNG)
![edge-screen-1](https://user-images.githubusercontent.com/313477/57849855-1b416f00-77e5-11e9-9292-07e43c6caacc.jpeg)
![edge-version](https://user-images.githubusercontent.com/313477/57849857-1d0b3280-77e5-11e9-8005-b32da380ee32.jpeg)

**Comment by mymindstorm** [comment]
What's the extension version? You're probably on an older version of the extension. This isn't a bug.

**Comment by netandreus** [comment]
@mymindstorm Extension version is 5.3.1 I installed it from Microsoft store yesterday. Why it isn't a bug? If true - why do yo make different icons for the same button?

**Comment by mymindstorm** [comment]
I forgot why. We changed the scan icon a few versions ago. I either had to make a few patches to the edge version to make it work properly and couldn't get the new scan icon to work, or I only updated the translations.

Everything in Edge is painful since the devtools don't even work most of the time and the fact that it's really buggy. I actually got an email from a Microsoft contractor a while back saying that the entire scan function broke in an insider build for some reason. This reminds me that I should check with them to see if it was fixed yet. Every time I have to throw together an Edge release it takes me hours to test and make changes for it. I'm honestly just considering pulling the edge build from the store at this point.
